models/_sector/nft/trades/nft_ethereum_trades_beta.sql

```diff
@@ -25,6 +25,7 @@
     ,('collectionswap',    'v1',   ref('collectionswap_ethereum_base_trades'))
     ,('looksrare',    'v1',   ref('looksrare_v1_ethereum_base_trades'))
     ,('looksrare',    'v2',   ref('looksrare_v2_ethereum_base_trades'))
+    ,('looksrare',    'seaport',   ref('looksrare_seaport_ethereum_base_trades'))
```

Comment:
Still a bit on the fence if we should keep the `seaport` version name, but not blocking this **clean** PR for that discussion. We can easily change it later if we want.
cc: @jeff-dude